content/docs/command-reference/exp/init.md

```diff
@@ -21,8 +21,10 @@ usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
 `dvc exp init` helps you get started with DVC Experiments quickly. It reduces
```

Comment:
Suggested rewording:

> It creates an experiment pipeline in a `dvc.yaml` file, which assumes standard locations for the inputs (source code, data, and parameters) and outputs (model, metrics, and plots) of your model training command. It also generates the input locations if they don't exist.

Reply:
I like this @dberenbaum . I would make some edits though. Let's move it to a separate PR I think... ⌛